### PR TITLE
Try: add max-height to query pattern preview

### PR DESCRIPTION
**Pull request by kjellr**
The query block's preview height is really large in Safari. I'm not sure what the max-height should be, but it should have one. 

From what I can tell, patterns shown in the sidebar don't have a max-height, but I think this is a different case — it'll look much cleaner if each item in the row has a shared height.

Before|After
---|---
Screen Shot 2021-04-12 at 11 32 26 AM|Screen Shot 2021-04-12 at 11 41 30 AM

### CODE REVIEW

File: packages/block-library/src/query/editor.scss [hunk 1]
```diff
@@ -31,6 +31,7 @@
 		cursor: pointer;
 		margin: 0 $grid-unit-15 $grid-unit-15 0;
 		width: 200px;
+		max-height: 200px;
```
Comment (kjellr) [inline packages/block-library/src/query/editor.scss]: I chose this value mostly at random — @jasmussen or @jameskoster, do you have a better idea for what this should be?

Comment (jameskoster) [reply packages/block-library/src/query/editor.scss]: Hmm, are we sure this isn't a bug somewhere in how these patterns are rendered? I don't see any reason for the thumbnails to be so tall. 🤔

Comment (kjellr) [reply packages/block-library/src/query/editor.scss]: Yeah, after I opened this I saw that it only happens in Safari, so I wonder if there's a browser thing we're not accounting for. 🤔

Comment (jasmussen) [reply packages/block-library/src/query/editor.scss]: Yeah I'm also seeing weirdness here and assuming a max-height. I could swear this used to work without one. I wonder what regressed.

**Comment by jasmussen** [comment]
I created a separate PR in https://github.com/WordPress/gutenberg/pull/30789, and then closed it and pushed the changes here instead. I hope you'll forgive me for that 😄  — essentially I realized it was the same work, so no reason to fragment the conversation. Now this PR also fixes the squashed "Start Blank" button:



Ultimately we should not optimize for ages on this, as I would expect the paginated version Nik has been working on as the replacement for this.

**Review by jasmussen — APPROVED** [review]
This is solid, thanks Kjell.

**Comment by ntsekouras** [comment]
There is as difference of heights in the first row.

### Before
Screenshot 2021-04-13 at 10 36 55 AM

### This PR

Screenshot 2021-04-13 at 10 37 10 AM

**Comment by jasmussen** [comment]
Oh great catch. Fixed:

Screenshot 2021-04-13 at 09 56 57

Yes, it's still not ideal how it wraps, but as noted I think the carousel should be the solution to that.

**Review by ntsekouras — APPROVED** [review]

**Comment by kjellr** [comment]
Awesome, thank you all for helping follow up on this. Looks good on my end!

**Pull request merged**